crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -470,7 +471,15 @@ def get_nodes_and_groups(node_name, draggable = true)
         @sum = @sum + node.name.hash
         @nodes[node.handle] = { alias: node.alias, description: node.description, status: node.status, state: node.state }
         group = node.group
-        @groups[group] = { automatic: !node.display_set?("group"), status: {"ready"=>0, "failed"=>0, "unknown"=>0, "unready"=>0, "pending"=>0}, nodes: {} } unless @groups.key? group
+        @groups[group] = { automatic: !node.display_set?("group"),
+                           status: {"ready"=>0,
```

Comment:
Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)<br>Style/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)